data/saas/dataset/segment_dataset.yml

```diff
@@ -81,41 +79,55 @@ dataset:
             data_categories: [user]
           - name: email
             data_categories: [user.contact.email]
-            fidesops_meta:
+            fides_meta:
               data_type: string
           - name: firstName
             data_categories: [user.name]
-            fidesops_meta:
+            fides_meta:
+              data_type: string
+          - name: first_name
+            data_categories: [user.name]
+            fides_meta:
               data_type: string
           - name: gender
             data_categories: [user.demographic.gender]
-            fidesops_meta:
+            fides_meta:
               data_type: string
           - name: id
             data_categories: [user.unique_id]
+            fides_meta:
+              primary_key: true
```

Comment:
This is the spot @galvana - happy to try and move it elsewhere if a different space makes sense? I think I was also looking at having this be where the erase happened instead of in the data protection request

Reply:
Ah, I completely missed it, this makes sense though 👍